src/client/datascience/kernel-launcher/kernelFinder.ts

```diff
@@ -27,7 +27,7 @@ const linuxJupyterPath = path.join('.local', 'share', 'jupyter', 'kernels');
 const macJupyterPath = path.join('Library', 'Jupyter', 'kernels');
 const baseKernelPath = path.join('share', 'jupyter', 'kernels');
 
-const cacheFile = 'kernelSpecPathCache.json';
+const cacheFile = 'kernelSpecPaths.json';
```

Comment:
The contents of the previous file are no longer valid, hence creating a new file.

Reply:
@IanMatthewHuff /cc

Reply:
Thanks!